.travis.yml

```diff
@@ -146,7 +146,9 @@ before_script:
 
 # Tests running commands
 script:
-  - "[ $TESTS != js ] || make frontend_install_tests"
+  # make frontend_install_tests takes 17 min, so we only do it post submit
+  - "[ $TESTS = js -a ${TRAVIS_PULL_REQUEST} != false ] && make frontend || false"
```

Comment:
This should use the `[ ... ] || ...` style of the other commands. Otherwise, the command just always fails.